virttest/env_process.py

```diff
@@ -901,6 +901,21 @@ def postprocess(test, params, env):
     """
     error_context.context("postprocessing")
     err = ""
+    vm = env.get_vm(params.get('main_vm'))
+    if params.get("verify_guest_dmesg", "no") == "yes" and vm.is_alive():
+        try:
+            session = vm.wait_for_login()
+        except:
+            logging.warning("Could not get VM session to verify dmesg")
+        else:
+            guest_dmesg_log_file = params.get("guest_dmesg_logfile", "guest_dmesg.log")
+            level = params.get("guest_dmesg_level", 3)
+            ignore_result = params.get("guest_dmesg_ignore", "no") == "yes"
+            dmesg_log_file = utils_misc.get_path(test.debugdir, dmesg_log_file)
+            utils_misc.verify_dmesg(dmesg_log_file=dmesg_log_file,
```

Comment:
@balamuruhans both are RFC patchs, not sure if we need both of them yet, moreover it is better to use `utils_misc.verify_dmesg()` directly for arguments flexibility, etc.